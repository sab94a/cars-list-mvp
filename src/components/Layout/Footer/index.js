//@flow

import React from 'react';
import type { Node } from 'react';
import styles from './index.module.scss';

type Props = {
    children: Node
}

const Footer = ({ children }: Props) => (
    <footer className={ styles.root }>{ children }</footer>
);

export default Footer;
