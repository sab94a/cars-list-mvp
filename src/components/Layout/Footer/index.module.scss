@import 'components/lib/variables.scss';

.root {
    display: flex;
    border-top: $border;
    height: $panel-height + $border-width;
    align-items: center;
    justify-content: center;
}
