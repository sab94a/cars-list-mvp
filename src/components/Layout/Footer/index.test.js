import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './';

describe('<Footer>', () => {
    let component = null;
    const text = 'Footer';

    beforeEach(() => {
        component = shallow(<Footer>{ text }</Footer>);
    });

    it('Should render correctly', () => {
        expect(toJson(component)).toMatchSnapshot();
    });

    it('Should have right footer text', () => {
        expect(component.text()).toEqual(text);
    });
});
