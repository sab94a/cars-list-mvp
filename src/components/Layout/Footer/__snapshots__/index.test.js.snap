// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> Should render correctly 1`] = `
<footer
  className="root"
>
  Footer
</footer>
`;
